scripts/protocgen.sh

```diff
@@ -2,23 +2,16 @@
 
 set -eo pipefail
 
-proto_dirs=$(find ./proto -path -prune -o -name '*.proto' -print0 | xargs -0 -n1 dirname | sort | uniq)
-for dir in $proto_dirs; do
-  buf protoc \
-  -I "proto" \
-  -I "third_party/proto" \
-  --gocosmos_out=plugins=interfacetype+grpc,\
-Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types:. \
-  $(find "${dir}" -maxdepth 1 -name '*.proto')
+echo "Generating gogo proto code"
+cd proto
 
-  # command to generate gRPC gateway (*.pb.gw.go in respective modules) files
-  buf protoc \
-  -I "proto" \
-  -I "third_party/proto" \
-  --grpc-gateway_out=logtostderr=true:. \
-  $(find "${dir}" -maxdepth 1 -name '*.proto')
+buf generate --template buf.gen.gogo.yaml
 
-done
+cd ..
 
+# move proto files to the right places
 cp -r github.com/datachainlab/ethereum-ibc-relay-prover/* ./
+cp -r github.com/datachainlab/ethereum-lcp/* ./
```

Comment:
This addition is not needed if the proto of light client definition is copied from ethereum-ibc-rs.

Reply:
Remove cp command
https://github.com/datachainlab/ethereum-ibc-relay-prover/pull/3/commits/7484802a5ca3f65de988f15af8808db21dce92be